packages/add-inbox/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@novu/add-inbox",
+  "version": "1.0.0",
+  "description": "Add inbox notifications to your application with one command",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/novu/novu.git"
+  },
+  "homepage": "https://novu.co",
+  "bugs": {
+    "url": "https://github.com/novu/novu/issues"
+  },
+  "license": "ISC",
+  "author": "Novu",
+  "type": "commonjs",
+  "main": "index.js",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "tsc",
+    "start": "ts-node src/cli/index.ts",
+    "prepublishOnly": "npm run build"
+},
+"keywords": [
```

Comment:
this file needs to be prettyfied, make sure that others are also run with prettier